src/sentry/logging/handlers.py

```diff
@@ -48,13 +48,23 @@ def __call__(self, logger, name, event_dict):
 
 class StructLogHandler(logging.StreamHandler):
     def emit(self, record, logger=get_logger()):
+        # This may seem a little gross, but when logging with the RootLogger,
+        # anything passed into 'extra' gets turned into a real kwarg
+        throwaways = [
```

Comment:
It's all of the attributes in a `logging.LogRecord` that we don't need in the event dictionary.